include/wimlib/file_io.h

```diff
@@ -11,27 +11,27 @@
 struct filedes {
 	int fd;
 	unsigned int is_pipe : 1;
-	off_t offset;
```

Comment:
You can avoid this whole patch by adding the following the the preprocessor definitions:
```
_OFF_T_DEFINED;_off_t=__int64;off_t=_off_t;_FILE_OFFSET_BITS=64;
```